certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1417 @@
+                              "description": "additionalProp2"
+                            },
+                            "additionalProp3": {
+                              "type": "string",
+                              "description": "additionalProp3"
+                            }
+                          },
+                          "description": "regionalPermId"
+                        }
+                      },
+                      "description": "idResolutions"
+                    },
+                    "dateRange": {
+                      "type": "object",
+                      "properties": {
+                        "created": {
+                          "type": "object",
+                          "properties": {
+                            "gt": {
+                              "type": "string",
+                              "description": "gt"
+                            }
+                          },
+                          "description": "created"
+                        },
+                        "updated": {
+                          "type": "object",
+                          "properties": {
+                            "gt": {
+                              "type": "string",
+                              "description": "gt"
+                            }
+                          },
+                          "description": "updated"
+                        }
+                      },
+                      "description": "dateRange"
+                    }
+                  },
+                  "description": "meta"
+                },
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "title"
+                      },
+                      "detail": {
+                        "type": "string",
+                        "description": "detail"
+                      },
+                      "source": {
+                        "type": "object",
+                        "properties": {
+                          "parameter": {
+                            "type": "string",
+                            "description": "parameter"
+                          }
+                        },
+                        "description": "source"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Signals - Headlines",

```

Comment:
@krithsai A summary can be a one-line sentence that would explain the operation. The provided summary value of two words doesn't seem a valid one, please update it with a valid summary value. This comment applies to all the summary values in the swagger.

Reply:
This is an appropriate value that describes the operation. Since our client services folks will be involved in satisfying the pre-requisite access requirements for the connector, I do believe this value is sufficient and can be explained well to our client base.